docs/source/format/Columnar.rst

```diff
@@ -622,11 +619,11 @@ use cases:
 * A sparse union is more amenable to vectorized expression evaluation in some use cases.
 * Equal-length arrays can be interpreted as a union by only defining the types array.
 
-**Example layout: ``SparseUnion<u0: Int32, u1: Float, u2: VarBinary>``**
+**Example layout: ``SparseUnion<i: Int32, f: Float, s: VarBinary>``**
```

Comment:
While we are at it, can we use consistent casing for types (here we use Int32 and Float, elsewhere it's lowercased, and the examples seem a bit of a mixed bag :-))?

Reply:
will do